Core/StitchCoreSDK/Sources/StitchCoreSDK/Internal/Net/AppMetadata.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+/// Application Metadata
+public final class AppMetadata: Codable, CustomStringConvertible {
```

Comment:
[design] On second thought, this could be a `struct`: https://blog.usejournal.com/swift-basics-struct-vs-class-31b44ade28ae Regardless of whether it being a value vs. a reference affects our current intended use, I feel that `struct` as a keyword has an implication to the reader of what the structure may or may not be used for.

Reply:
done